### DISCUSSION TITLE
How much GPU memory is needed for finetuning stt_en_conformer_ctc_small?

### DISCUSSION THREAD

**Discussion by tareqalmuntasir7** [NVIDIA-NeMo/NeMo#3252] [Q&A] [answered]
Hi, I am trying to finetune stt_en_conformer_ctc_small, both encoder and decoder with a RTX 3090 24GB GPU. But even when I use batch size 1, it's throwing OOM. Is this model too big or is it a bug? I see there are only 51 million parameters in the model.

**Comment by titu1994** [comment, marked as answer]
What is your maximum audio length ? It should not be more than 16-20 seconds during training. We can fit about batch size 16 with 32 GB of RAM at fp32, and bs 32 at fp16 (but training is unstable with fp16)
